web/assets/js/hotjar.js

```diff
@@ -0,0 +1,8 @@
+(function(h,o,t,j,a,r){
+    h.hj=h.hj||function(){(h.hj.q=h.hj.q||[]).push(arguments)};
```

Comment:
To avoid issues when env is not set (`hotjar.js:14 
GET https://static.hotjar.com/c/hotjar-undefined.js?sv=6 net::ERR_ABORTED 404
`):

```suggestion
(function(h,o,t,j,a,r){
    let hotjarId = process.env.HOTJAR_ID;
    if (typeof(hotjarId) === 'undefined') { return; }
    h.hj=h.hj||function(){(h.hj.q=h.hj.q||[]).push(arguments)};
    h._hjSettings={hjid:hotjarId,hjsv:6};
    a=o.getElementsByTagName('head')[0];
    r=o.createElement('script');r.async=1;
    r.src=t+h._hjSettings.hjid+j+h._hjSettings.hjsv;
    a.appendChild(r);
})(window,document,'https://static.hotjar.com/c/hotjar-','.js?sv=');
```